Contents/mods/More Traits/media/lua/client/MoreTraits.lua

```diff
@@ -1310,19 +1320,28 @@ local function amputee(_player, justGotInfected)
         if UpperArm_L:HasInjury() then
             UpperArm_L:RestoreToFullHealth();
             if justGotInfected then
-                player:getBodyDamage():setInfected(false);
+                bodydamage:setInfected(false);
+                bodydamage:setInfectionMortalityDuration(-1);
+                bodydamage:setInfectionTime(-1);
+                bodydamage:setInfectionLevel(0);
```

Comment:
have we been able to determine that setting this value will prevent Infection Timer from growing?

Reply:
Well, `setInfectionTime(-1);` stops infection timer and therefore prevent it from growing. If you want to save current infection level, you can remove `setInfectionLevel(0);`, but I did not explicitly test that, as previous code did also reset infection level.

Reply:
Sounds good enough for me.